wolfssl/wolfcrypt/types.h

```diff
@@ -125,7 +125,12 @@
         WOLFSSL_WORD_BITS  = WOLFSSL_WORD_SIZE * WOLFSSL_BIT_SIZE
     };
 
-    #define WOLFSSL_MAX_16BIT 0xffffU
+    #define WOLFSSL_MAX_BYTE   0xffU
+    #define WOLFSSL_MAX_16BIT  0xffffU
+    #define WOLFSSL_MAX_32BIT  0xffffffffU
```

Comment:
These are not the right types.  See sha256 and sha512 for example on the right types.

Reply:
Could you clarify what you mean by "not the right types".

I was defining maximum values that could be stored in our fixed-width types byte, word16, word32, and if available word64, not defining new types.

Reply:
There needs to be a way to turn these off, better yet.  Until we know that they work with 100% of our environments let's have them be off by default with a define to enable them.